Source/Flow/Public/Interfaces/FlowDataPinGeneratorNodeInterface.h

```diff
@@ -0,0 +1,29 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "UObject/Interface.h"
+#include "Nodes/FlowPin.h"
+
+#include "FlowDataPinGeneratorNodeInterface.generated.h"
+
+// Interface for special UFlowNodes that can auto-generate pins directly
+UINTERFACE(MinimalAPI, NotBlueprintable, DisplayName = "Flow Data Pin Generator Node Interface")
+class UFlowDataPinGeneratorNodeInterface : public UInterface
+{
+	GENERATED_BODY()
+};
+
+class FLOW_API IFlowDataPinGeneratorNodeInterface
+{
+	GENERATED_BODY()
+
+public:
+
+#if WITH_EDITOR
```

Comment:
Only a few flow nodes are DataPin generators, so I left this to its own interface, rather than building the function into UFlowNode.